### PR TITLE
Quick code lint

### PR DESCRIPTION
**Pull request by Mervill**
Das ist verboten!!

### CODE REVIEW

File: code/modules/power/port_gen.dm [hunk 1]
```diff
@@ -332,8 +332,9 @@ display round(lastgen) and plasmatank amount
 	power_gen = 15000
 	time_per_sheet = 85
 	board_path = /obj/item/weapon/circuitboard/machine/pacman/super
-	overheat()
-		explosion(src.loc, 3, 3, 3, -1)
```
Comment (Cyberboss) [inline code/modules/power/port_gen.dm]: How the hell did this even parse?

Comment (Cyberboss) [inline code/modules/power/port_gen.dm]: Oh wait, indent style... so shitty

**Comment by kevinz000** [comment]
Isn't this how bay does their stuff?

**Comment by RemieRichards** [comment]
@kevinz000 yes and it's dumb.

**Review by phil235 — APPROVED** [review]
👍

**Pull request merged**